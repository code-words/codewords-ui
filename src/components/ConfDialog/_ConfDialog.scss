.overlay {
  position: absolute;
  z-index: 499;
  top: 0;
  left: 0;
  background: rgba(0, 0, 0, 0.091);
  height: 100vh;
  width: 100vw;
  display: flex;
  align-items: center;
  justify-content: center;
}

.dialog {
  height: 200px;
  width: 600px;
  background: rgba(255, 255, 255, 0.975);
  color: rgb(45, 45, 45);
  border: 2px dashed rgb(131, 131, 131);
  font-size: 32px;
  z-index: 700;
  border-radius: 15px;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;

  .replay-btns {
    display: flex;
  }
}