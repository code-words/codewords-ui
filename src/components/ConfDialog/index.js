import React from 'react';
import { Link } from 'react-router-dom'

const ConfDialog = props => {
  const buttons = !props.over ? null
    : <div className="replay-btns">
        <Link to="/" className="nav-item btn-default start-new-btn">
          Start New Game
        </Link>
        <Link to="/" className="nav-item btn-default replay-btn">
          Play Again!
        </Link>
      </div>

  return (
    <div className="overlay">
      <div className="dialog">
        {props.message}
        {buttons}
      </div>
    </div>
  );
};

export default ConfDialog;