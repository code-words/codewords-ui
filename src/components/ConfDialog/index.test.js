import ConfDialog from './index'
import React from 'react'
import { shallow } from 'enzyme'

describe('ConfDialog', () => {
    let wrapper = shallow(<ConfDialog/>)
    it('should match snapshot', () => {
        expect(wrapper).toMatchSnapshot()
    });
});