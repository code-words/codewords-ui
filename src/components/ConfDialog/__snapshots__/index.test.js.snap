// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfDialog should match snapshot 1`] = `ShallowWrapper {}`;
